### DISCUSSION TITLE
Allow Umbraco Tags within Block List property editor

### DISCUSSION THREAD

**Discussion by bkclerke** [umbraco/Umbraco-CMS#12248] [Features and ideas]
Can we get the ability to use the Umbraco Tags property editor within the Block List property editor?
![image (2)](https://user-images.githubusercontent.com/4968254/162847005-2a00ad0e-db4a-4d29-a626-72a3c2982bad.png)

Currently, Umbraco does not support using tags in the block list editor, however, we have a need for this where we are listing stories and allowing the content editors to filter the stories returned based on what matches the categories they have specified. This tag editor is also used on the individual stories so the tag group functionality comes into play here and allows the editors to easily add filters based on pre-existing tags. 

This would be nice to have! Pretty please! :)

**Comment by skttl** [comment]
> Tags are disabled in element types, because the tag editor does much more than just adding some text labels in a specific property. When you save a node with tags, an index of tags and their related nodes are created, for you to search in. So as an example, you can find all nodes tagged with codegarden, by using a [TagQuery with the Umbraco Helper](https://our.umbraco.com/Documentation/Reference/Querying/UmbracoHelper/#working-with-tags). This requires the nodes to be actual nodes - which element type based nodes is not. They are just json blobs stored in your nodes, and the TagQuery can't find tags inside that.
> 
> Since the prevention is based on the property editor alias, all you need is a tag-editor with a non-standard alias.
 
Check this out https://dev.to/skttl/bypassing-property-editor-restrictions-in-umbraco-element-types-21ko
